bears/swift/TailorBear.py

```diff
@@ -0,0 +1,54 @@
+import json
+
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from coalib.results.Result import Result
+from coalib.settings.Setting import path
+
+
+@linter(executable="tailor",
+        output_format=None,
+        prerequisite_check_command=("tailor", "-v"),
+        prerequisite_check_fail_message="Tailor is not installed. Refer "
+        "https://tailor.sh/ for installation details.")
+class TailorBear:
+    """
+    Perform static analysis on Swift source files using Tailor.
```

Comment:
this still doesn't tell me what kind of analysis exactly it does... i think this is really only codestyle so can you be very concrete here and claim only codestyle analysis?